public/app/plugins/datasource/cloudwatch/types.ts

```diff
@@ -451,6 +451,11 @@ export interface LegacyAnnotationQuery extends MetricStat, DataQuery {
   type: string;
 }
 
+export interface MetricResponse {
```

Comment:
I must be missing something, is that the same as line 480? https://github.com/grafana/grafana/pull/57602/files#diff-d543b94a1c441fafa83a037b7a8738382323dfb41fcbc1e690c448999181f1f1R480

Reply:
Yeah good shout! Can't believe the linter did complain? Anyway, will fix